templates/taxi/car_list.html

```diff
@@ -5,6 +5,7 @@ <h1>
     Car list
     <a href="{% url 'taxi:car-create' %}" class="btn btn-primary link-to-page">
       Create
+    <a href="{% url 'taxi:driver-list' %}" class="btn btn-secondary">Cancel</a>
```

Comment:
The 'Cancel' button should lead to the previous page the user was on. Consider using JavaScript to achieve this, such as `onclick="history.back()"`.